.tekton/cli-main-ci-pull-request.yaml

```diff
@@ -26,10 +28,10 @@ spec:
       value: quay.io/redhat-user-workloads/rhtap-contract-tenant/ec-main-ci/cli-main-ci:on-pr-{{revision}}
     - name: bundle-cli-ref-repo
       value: quay.io/redhat-user-workloads/rhtap-contract-tenant/ec-main-ci/cli-main-ci
-    - name: image-expires-after
-      value: 5d
     - name: dockerfile
       value: Dockerfile.dist
+    - name: image-expires-after
+      value: 5d
```

Comment:
Do we need double quotes? I didn't think so, but noticed it in other places.

Reply:
I think it's okay.

```
echo 'value: 5d' | yq -ojson
{
  "value": "5d"
}
```